### DISCUSSION TITLE
聊天時的尷尬谷

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1558] [General]
人們在對話中的禮貌性停頓​
大約是 0.2 秒，比眨個眼（0.35 秒）還要短​
一旦超過這個時間，雙方就容易感到不對勁​，讓某一方開始尋思「我是不是該接話了？」​
「他掉線了嗎？還要繼續說下去嗎？」​

但不只要避免停太久，也切記不要講太多​
比如實驗發現，某些小鳥在對話的時候​
如果有一方搶快、害彼此的聲音不小心重疊​
雙方頓時會陷入沉默、讓空氣突然變得安靜

**Comment by aa103967622** [comment]
講話的時候，如果無法及時接話就會陷入尷尬，這可能是我們日常生活中常遇到的情況，我的印象中，這樣的情況通常發生於你不那麼熟識的對象，或是跟你價值觀不太合的說話對象，因為你們缺乏共同話題，以及那種即使不說話也不會尷尬的友好程度。

**Comment by ThanatosJun** [comment]
「說話是種藝術」，這句話很多人愛說，現在我又說了一次。我認為「說話」是可以勉強的，依照每個人不同的個性，對與陌生人說話這件事各有不同的反應，但對談仍可持續，這是為什麼？或許是有一方在勉強，或許兩方都在勉強呢！

**Comment by Alex11392** [comment]
好有趣的分享。原來禮貌性停頓的間隔這麼短嗎?我本身也是很容易陷入尷尬的人，不知道有沒有甚麼好方法可以更好延續話題

**Comment by lin157644** [comment]
很有趣的研究，不知道這種尷尬或不對勁的感覺要怎麼量化:smile:，我自己有時候一時語塞或是不知該如何接話時，也靠一些 痾~阿~，一類的語助詞來還解尷尬，會講話也是一門才藝，那些不管什麼時候總能侃侃而談的人實在是很厲害。

**Comment by BOSENCHANG2004** [comment]
尬谷其實像是對話的「低谷」，但有時候跨過去，反而能讓彼此更放鬆。
像朋友之間的尷尬，有時候只是熟悉感的醞釀。

**Comment by SendaiHatsuki** [comment]
0.2 秒根本就是反射等級的反應了，難怪有時候和人講話只要停個一兩秒，就會瞬間尷尬，對方可能就開始懷疑是不是該換人說話了。這也讓我想到，有些人講話節奏比較慢，就容易在聊天時被誤會成「冷場製造機」，其實只是還在組織語句。相反地，如果搶話或不斷講不停，雖然話題一直有在進行，卻可能讓人喘不過氣，甚至沒辦法插話，變成單方面的輸出會讓人感覺壓迫感很重。
我覺得這真的很看天分，有些人就是那種「很會抓節奏」的跟他講話總是覺得超流暢、不會尷尬，又很輕鬆。

**Comment by edawdewv** [comment]
有時候明明想聊點什麼，但話題突然沒了，或對方回應很短，整個空氣像凝固一樣。然後就開始尷尬地笑笑，或一直低頭滑手機。這種時候其實不用太緊張，反正誰都有遇過。可以試著換個輕鬆話題，或問個簡單的問題像「最近有什麼好看的劇嗎？」這樣比較容易接話。或者乾脆笑說「欸，怎麼突然安靜了？」反而會讓氣氛活起來。

**Comment by alice20040127** [comment]
感覺0.2秒真的是太短了，身為其實不是很擅長跟其他人主動說話或開啟話題的I人，有時候真的不是不想回話，只是還在思考要回什麼才能繼續延續話題，要立即反應出來然後又不是句點的話感覺真的很難，再加上有時候剛認識也會有所顧慮，會有些話題是不太敢去開口的，於是能選擇的話題就變得更少了，思考時也是壓力很大。

**Comment by wuyu06** [comment]
這篇講聊天裡那種尷尬的停頓超有感 原來人們連0.2秒的空白都覺得怪怪的，真的會讓人不知道該不該接話，好像對方突然消失一樣
我覺得溝通真的是很講節奏感的事，聊天時多注意這點，應該能避免很多尷尬

**Comment by mattwang1222** [comment]
這種「尷尬谷」其實很真實，我自己也常在對話時因停太久或搶話而感到不自在。溝通其實就像跳舞，需要節奏和默契。比起話多，不如學會適時停頓，讓彼此都有喘息與回應的空間。
